### DISCUSSION TITLE
Need Help with a Scheduled Task to Restart Explorer

### DISCUSSION THREAD

**Discussion by DaveTheNoob** [valinet/ExplorerPatcher#1657] [General]
Thank you for developing this!!!

My only "issue" is that about once or twice a day, the taskbar disappears on one screen ( I have 4) and I just have to click on  "Restart File Explorer" link in the Properties box. 

I'm assuming that I can just write a scheduled task that will restart Explorer for me automatically at the times I set. 

I'm just wondering what's the service or .exe name I need to target in my Scheduled Task to do this? 

Other than this, it's been absolutely flawless for me. Here's my OS info (in case it helps). 

Edition	Windows 11 Pro
Version	22H2
Installed on	‎2/‎20/‎2023
OS build	22621.1992
Experience	Windows Feature Experience Pack 1000.22644.1000.0

**Comment by pyrates999** [comment]
You would create the scheduled task with elevated privileges and kill the process: C:\Windows\explorer.exe

But before you do that, uninstall explorer patcher and see if it still happens.  If it does, you may need to repair your windows install using sfc and dism:

https://www.computerworld.com/article/3080839/repair-windows-10-and-11-step-by-step-guide.html
